lib/taurus/core/tango/tangodatabase.py

```diff
@@ -679,8 +679,12 @@ def __init__(self, host=None, port=None, parent=None):
 
         # Set host to fqdn
         host = socket.getfqdn(host)
+        try:
+            self.dbObj = Database(host, port)
+        except Exception as e:
+            from taurus import warning
+            warning(e)
```

Comment:
IMHO, the warning message should be a bit more descriptive. For example:
`warning("Cannot connect with Tango DB: %r", e)

Reply:
I change it